app/models/miq_report.rb

```diff
@@ -181,6 +181,17 @@ def sort_col
     sortby ? col_order.index(sortby.first) : 0
   end
 
+  def column_is_hidden?(col)
+    return false unless col_options
+
+    @hidden_cols ||= col_options.keys.each_with_object([]) do |c, a|
+      a << c if col_options[c][:hidden]
+      a
```

Comment:
You do not need the array as the last line in the block when using `each_with_object`.  This line can be removed.

Reply:
@gmcculloug I was going to remove that but then I realized that if the column is not hidden, the last line in the block would be nil. Not sure if that's correct. I'll need to try it. Thanks.